src/debug_info.js

```diff
@@ -120,7 +120,7 @@ class DebugInfo {
 
                     if (statement.symbol) {
                         debugInfo.symbols.push({
-                            name: statement.symbol,
+                            name: statement.symbol.replace(':',''),
```

Comment:
Fix is ok. Maybe for a future improvement, string replacing should be replaced by better parsing.

Reply:
Yes I agree. Will look into that some day.